notebooks/feature_serving_batch.py

```
@@ -0,0 +1,68 @@
+# Databricks notebook source
+# MAGIC %pip install /Volumes/marvelous_dev_ops/personality_types/package/mlops_with_databricks-0.0.1-py3-none-any.whl --force-reinstall
```

Ruff

Line too long (136 > 79)

(E501)